rules/repositories.bzl

```diff
@@ -27,15 +27,15 @@ def rules_ios_dependencies():
     _maybe(
         git_repository,
         name = "build_bazel_rules_apple",
-        commit = "fe8cf69b058c86624ba1786e6024c2a1af1d48e9",
+        commit = "32ede960cb9029be15890622366eb5598ceebc74",
         shallow_since = "1581368998 -0800",
         remote = "https://github.com/bazelbuild/rules_apple.git",
     )
 
     _maybe(
         git_repository,
         name = "build_bazel_rules_swift",
-        commit = "657eda57a097980848dbb3c880b94faeddaa3cd1",
```

Comment:
`DEBUG: Rule 'build_bazel_rules_swift' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1586818875 -0700"`